docs/ref/models/database-functions.txt

```diff
@@ -175,8 +144,67 @@ will result in a database error.
     The PostgreSQL behavior can be emulated using ``Coalesce`` if you know
     a sensible maximum value to provide as a default.
 
+``Now``
+=======
+
+.. class:: Now()
+
+Returns the database server's current date and time when the query is executed,
+typically using the SQL ``CURRENT_TIMESTAMP``.
+
+Usage example::
+
+    >>> from django.db.models.functions import Now
+    >>> Article.objects.filter(published__lte=Now())
+    <QuerySet [<Article: How to Django>]>
+
+.. admonition:: PostgreSQL considerations
+
+    On PostgreSQL, the SQL ``CURRENT_TIMESTAMP`` returns the time that the
+    current transaction started. Therefore for cross-database compatibility,
+    ``Now()`` uses ``STATEMENT_TIMESTAMP`` instead. If you need the transaction
+    timestamp, use :class:`django.contrib.postgres.functions.TransactionNow`.
+
+.. _text-functions:
+
+Text Functions
+==============
+
+.. module:: django.db.models.functions.text
```

Comment:
I think we shouldn't document the functions in the "text" namespace. Most other references in Django are documented in the convenient import location. Not doing that for functions.datetime was probably a mistake.